manifests/web.pp

```diff
@@ -377,6 +401,47 @@
   # Is set to true, it will create the apache vhost.
   if $manage_vhost {
     include apache
+    if facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'], '7') == 0 and versioncmp($zabbix_version, '5') >= 0 {
```

Comment:
can you add an acceptance test for zabbix 5 to cover this? If you've questions about the tests or beaker, feel free to ping us on IRC in #voxpupuli on freenode.

Reply:
Can do.

I have another pull request I was working on but hadn't submitted yet to resolve a specific issue related to a zabbix_proxy.conf parameter needing to be added to the template to allow overriding of the parameter to resolve an issue that occurred after upgrading to Zabbix 5 where the zabbix-proxy service would either fail to start correctly or start and then immediately terminate due to if I remember correctly selinux permissions.

I assume that not including those additional changes would cause a failure in the beaker tests for CentOS 7/RHEL 7 for a proxy node when selinux is enabled.

Should I merge those changes into this pull request, or keep it as a separate pull request and exclude that specific failing scenario from the acceptance tests?

Reply:
I think I answered my own question by reviewing the acceptance tests; I can update server_spec.rb within this PR and include an updated zabbix_proxy_spec.rb within the other PR.